tests/integration/admin_test.py

```diff
@@ -80,6 +81,89 @@ def test_retrieve_all_groups(client: AtlanClient, group: CreateGroupResponse):
             _default_group_count += 1
 
 
+def test_group_get_pagination(client: AtlanClient, group: CreateGroupResponse):
+    response = client.group.get(limit=1)
+
+    assert response
+    assert response.total_record is not None
+    assert response.total_record >= 1
+    current_page = response.current_page()
+    assert current_page is not None
+    assert len(current_page) >= 1
+    for test_group in response:
+        assert test_group.id
+        assert test_group.name
+        assert test_group.path
+        assert test_group.attributes
+    current_page = response.current_page()
+    assert current_page is not None
+    assert len(current_page) == 0
+
+
+def test_group_get_members_pagination(client: AtlanClient, group: CreateGroupResponse):
+    groups = client.group.get_by_name(alias=GROUP_NAME)
+    assert groups
+    assert len(groups) == 1
+    group1 = groups[0]
+    assert group1.id
+    response = client.group.get_members(guid=group1.id, request=UserRequest(limit=1))
+
+    assert response
+    assert response.total_record is not None
+    assert response.total_record >= 1
+    current_page = response.current_page()
+    assert current_page is not None
+    assert len(current_page) >= 1
+    for test_user in response:
+        assert test_user.username
+        assert test_user.email
+        assert test_user.attributes
+    current_page = response.current_page()
+    assert current_page is not None
+    assert len(current_page) == 0
+
+
+def test_user_list_pagination(client: AtlanClient, group: CreateGroupResponse):
+    response = client.user.get(limit=1)
+
+    assert response
+    assert response.total_record is not None
+    assert response.total_record >= 1
+    current_page = response.current_page()
+    assert current_page is not None
+    assert len(current_page) >= 1
+    for test_user in response:
+        assert test_user.username
+        assert test_user.email
+        assert test_user.attributes
+        assert test_user.login_events
+    current_page = response.current_page()
+    assert current_page is not None
+    assert len(current_page) == 0
```

Comment:
Won't this still succeed if there is only a single user result across all pages?

May be difficult with our current setup for groups (we only have 1 group by default); but for users we should be checking that the total number is strictly greater than (`>`) 1, not gte (`>=`) I think.

Reply:
Thanks @cmgrote!
For the user test, `response.total_record` should be `> 1`. I think we should also change `len(current_page) >= 1` to `len(current_page) == 1` in all pagination tests, given that the **limit** is set to **1** 👍